src/ampl/meson.build

```diff
@@ -2,4 +2,6 @@ if libampl.found()
   libgalahad_ampl_src += files('galahad_ampl.c', 'wrap_bqp.F90', 'wrap_cqp.F90',
                                'wrap_filtrane.F90', 'wrap_lancelot.F90', 'wrap_presolve.F90',
                                'wrap_qp.F90', 'wrap_qpa.F90', 'wrap_qpb.F90', 'wrap_qpc.F90')
+
+  galahad_headers += files('amplinter.h', 'galahad_ampl.h')
```

Comment:
Headers that we will install only if we only compile GALAHAD with the `AMPL` interface.